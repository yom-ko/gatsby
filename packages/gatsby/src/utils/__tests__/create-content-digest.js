const createContentDigest = require(`../create-content-digest`)

describe(`Create content digest`, () => {
  it(`returns the content digest when the input is a string`, () => {
    const input = JSON.stringify({ id: 1 })

    const contentDigest = createContentDigest(input)

    expect(contentDigest).toMatchSnapshot()
  })

  it(`returns the content digest when the input is a non string`, () => {
    const input = { id: 1 }

    const contentDigest = createContentDigest(input)

    expect(contentDigest).toMatchSnapshot()
  })
})
