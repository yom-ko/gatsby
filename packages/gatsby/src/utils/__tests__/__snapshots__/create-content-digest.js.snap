// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create content digest returns the content digest when the input is a non string 1`] = `"d2ce28b9a7fd7e4407e2b0fd499b7fe4"`;

exports[`Create content digest returns the content digest when the input is a string 1`] = `"d2ce28b9a7fd7e4407e2b0fd499b7fe4"`;
